<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registraturePanel</class>
 <widget class="QDialog" name="registraturePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_FIO">
        <property name="text">
         <string>ФИО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_age">
        <property name="text">
         <string>Возраст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_telephone">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_post">
        <property name="text">
         <string>Должность:   регистратор вет. учреждения</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Управление</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="button_registerPatient">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Зарегистрировать владельца</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_registerAnimal">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Зарегистрировать животного</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_registerVisit">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Записать на приём</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_changePassword">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сменить пароль</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
